src/index.js

```diff
@@ -5,48 +5,34 @@ const APIS = [
   {
     service: 'audioblocks',
     type: 'audio',
-    baseUrl: 'https://api.audioblocks.com/'
+    prefixUrl: 'https://api.audioblocks.com/api/v2/audio'
   },
   {
     service: 'videoblocks',
     type: 'video',
-    baseUrl: 'https://api.videoblocks.com/'
+    prefixUrl: 'https://api.videoblocks.com/api/v2/videos'
+  },
+  {
+    service: 'graphicstock',
+    type: 'image',
+    prefixUrl: 'https://api.graphicstock.com/api/v2/images'
   }
 ];
 
-const services = credentials =>
-  APIS.reduce(
-    (acc, { service, type, baseUrl }) => ({
-      ...acc,
-      [service]: new StoryblocksApi(baseUrl, credentials, [
-        { name: 'search', endpoint: e`/api/v1/stock-items/search` },
-        { name: 'categories', endpoint: e`/api/v1/stock-items/categories` },
-        // audio is the only endpoint with subcategories...
-        ...(type === 'audio'
-          ? [
-            {
-              name: 'subcategories',
-              endpoint: e`/api/v1/stock-items/categories/${'category'}/subcategories`
-            }
-          ]
-          : []),
-        { name: type, endpoint: e`/api/v1/stock-items/${'stock_item_id'}` },
-        {
-          name: 'similar',
-          endpoint: e`/api/v1/stock-items/similar/${'stock_item_id'}`
-        },
-        {
-          name: 'download',
-          endpoint: e`/api/v1/stock-items/download/${'stock_item_id'}/${'downloader_id'}`
-        },
-        { name: 'collections', endpoint: e`/api/v1/collections` },
-        {
-          name: 'collection',
-          endpoint: e`/api/v1/collections/${'collection_id'}`
-        }
-      ])
-    }),
-    {}
-  );
+function services (credentials) {
+  const services = {};
+  APIS.forEach(({ service, type, prefixUrl }) => {
+    services[service] = new StoryblocksApi(prefixUrl, credentials, [
+      { name: 'search', endpoint: e`search` },
+      { name: 'categories', endpoint: e`stock-item/categories` },
+      { name: type, endpoint: e`stock-item/details/${'stock_item_id'}` },
+      { name: 'similar', endpoint: e`stock-item/similar/${'stock_item_id'}` },
+      { name: 'download', endpoint: e`stock-item/download/${'stock_item_id'}` },
+      { name: 'collections', endpoint: e`collections` },
+      { name: 'collection', endpoint: e`collections/${'collection_id'}` }
+    ]);
+  });
+  return services;
+}
```

Comment:
```suggestion
function services (credentials) {
  const services = APIS.map(({ service, type, prefixUrl }) => [
    service,
    new StoryblocksApi(prefixUrl, credentials, [
      { name: 'search', endpoint: e`search` },
      { name: 'categories', endpoint: e`stock-item/categories` },
      { name: type, endpoint: e`stock-item/details/${'stock_item_id'}` },
      { name: 'similar', endpoint: e`stock-item/similar/${'stock_item_id'}` },
      { name: 'download', endpoint: e`stock-item/download/${'stock_item_id'}` },
      { name: 'collections', endpoint: e`collections` },
      { name: 'collection', endpoint: e`collections/${'collection_id'}` }
    ])
  ]));
  return new Map(services); // or Object.fromEntries(services)
}
```

Reply:
@timlesallen not sure either of those options is actually any easier to read?... and `Map` isn't the right thing in this case really...
```js
audioblocks.get('search')(...args);
```